lfs.c

```diff
@@ -411,6 +411,22 @@ static inline void lfs_superblock_tole32(lfs_superblock_t *superblock) {
     superblock->attr_max    = lfs_tole32(superblock->attr_max);
 }
 
+static inline bool lfs_mlist_isused(struct lfs_mlist *head,
```

Comment:
nit: I think this would be better as `lfs_mlist_isopen`, it is directly related to `lfs_file_open` and `lfs_dir_open`.

Reply:
its okay, changed it

Reply:
It looks like this is still `lfs_mlist_isused`, or am I missing a commit?

Reply:
sorry, updated it now :smile: